python/lsst/meas/deblender/baseline.py

```diff
@@ -675,23 +824,65 @@ def _fitPsf(fp, fmask, pk, pkF, pkres, fbb, peaks, peaksF, log, psf, psffwhm,
             img, varimg, psfChisqCut1, psfChisqCut2, psfChisqCut2b,
             tinyFootprintSize=2,
             ):
-    """!
+    """
     Fit a PSF + smooth background model (linear) to a small region around the peak.
-
-    @param[in]     fp            Footprint
-    @param[in]     fmask         the Mask plane for pixels in the Footprint
-    @param[in]     pk            the Peak within the Footprint that we are going to fit a PSF model for
-    @param[in]     pkF           the floating-point coordinates of this peak
-    @param[in,out] pkres         a PerPeak object that will hold the results for this peak
-    @param[in]     fbb           the bounding-box of this Footprint (a Box2I)
-    @param[in]     peaks         a list of all the Peak objects within this Footprint
-    @param[in]     peaksF        a python list of the floating-point (x,y) peak locations
-    @param[in,out] log           pex Log object
-    @param[in]     psf           afw PSF object
-    @param[in]     psffwhm       FWHM of the PSF, in pixels
-    @param[in]     img           the Image in which this Footprint finds itself
-    @param[in]     varimg        Variance plane
-    @param[in]     psfChisqCut*  floats: cuts in chisq-per-pixel at which to accept the PSF model
+    This routine uses a linear least squares algorithm to fit the peak (and neighboring peaks)
+    to the PSF.
+    To make the comparison more robust the peak is fit with and without including
+    additional terms to recenter the PSF.
+    If the peak matches a recentered PSF, a third fit is calculated by shifting the PSF to the new center.
+    See ``psfChisqCut*`` below for more.
+
+    Parameters
+    ----------
+    fp: `afw.detection.Footprint`
+        Footprint containing the Peaks to model
+    fmask: `afw.image.MaskU`
+        The Mask plane for pixels in the Footprint
+    pk: `afw.detection.PeakRecord`
+        The peak within the Footprint that we are going to fit with PSF model
+    pkF: `afw.geom.Point2D`
+        Floating point coordinates of the peak.
+    pkres: `meas.deblender.PerPeak`
+        Peak results object that will hold the results.
+    fbb: `afw.geom.Box2I`
+        Bounding box of ``fp``
+    peaks: `afw.detection.PeakCatalog`
+        Catalog of all peaks contained in ``fp``.
+    peaksF: list of `afw.geom.Point2D`
+        List of floating point coordinates of all of the peaks.
+    log: `log.Log`
+        LSST logger for logging purposes
+    psf: `afw.detection.Psf`
+        PSF of the image.
+    psffwhm: `float`
+        PSF FWHM in pixels
+    img: `afw.image.ImageF`
+        The image that contains the footprint.
+    varimg: `afw.image.ImageF`
+        The variance of the image that contains the footprint.
+    psfChisqCut*: `float`
```

Comment:
Oh no! Here it is again! 😆 

Given that both `_fitPsf` and `_fitPsfs` are private, I think one could refer to the other rather than duplicating the text.